src/views/modal/registration-js.php

```diff
@@ -62,7 +62,12 @@ class="tribe-tickets__item__attendee__fields__form <?php echo sanitize_html_clas
 					</h3>
 				</div>
 		<?php endforeach; ?>
-		<p class="tribe-tickets-notice tribe-tickets-notice--non-ar">
+		<p
+			class="tribe-tickets-notice tribe-tickets-notice--non-ar"
+			<?php if ( empty( $non_meta_count ) ) : ?>
+				style="display: none;"
```

Comment:
I thought inline `style="display:none"` was not recommended and we should be using a class instead, is that right?

Reply:
We're using hide/show on this - which adds/removes the inline style. I didn't want to make the notices hidden by default, and some of the classes for that are harder to override.

That said, open to suggestions.